examples/molecular_vae/README.md

```diff
@@ -0,0 +1,64 @@
+# Use PaccMann molecular VAE from a PaccMannRL inference pipeline
+
+### A short tutorial on training a molecular VAE to generate SMILES and subsequently use it for inference
+
+#### Training
+
+This trains a molecular VAE using encoder and decoder based on StackGRUs. For details, see the PaccMannRL papers, for [omics](https://www.sciencedirect.com/science/article/pii/S2589004221002376) and [proteins](https://iopscience.iop.org/article/10.1088/2632-2153/abe808/meta).
+
+```console
+conda activate gt4sd
+gt4sd-trainer  --training_pipeline_name paccmann-vae-trainer --epochs 250 --batch_size 4 --n_layers 1 --rnn_cell_size 16 --latent_dim 16 --train_smiles_filepath src/gt4sd/training_pipelines/tests/molecules.smi --test_smiles_filepath src/gt4sd/training_pipelines/tests/molecules.smi --model_path /tmp/gt4sd-paccmann-gp/ --training_name fast-example --eval_interval 15 --save_interval 15 --selfies
+*NOTE*: You might want to pass a SMILES/SELFIES language object via `--smiles_language_filepath`
```

Comment:
here I'm afraid the suggestion did not go through as expected, probably the three quotes are not being escaped

Reply:
fixed now